app/src/main/java/com/example/weather/CityPickerViewModel.kt

```diff
@@ -1,11 +1,39 @@
 package com.example.weather
 
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.weather.network.City
+import com.example.weather.network.WeatherApi
+import com.example.weather.network.WeatherApiResponse
+import kotlinx.coroutines.launch
 
 class CityPickerViewModel : ViewModel() {
 
-    private val _cities: List<Int> = (1..100).toList()
-    val cities: List<Int>
-    get() = _cities
+    private val _cities = MutableLiveData<List<City>>()
 
+    val cities: MutableLiveData<List<City>>
```

Comment:
Mentioned this before but this should be LiveData instead of MutableLiveData since we don't want the UI to be able to modify the viewModel's data